backtesting/backtesting.py

```diff
@@ -105,6 +105,10 @@ def I(self,  # noqa: E741, E743
         If `scatter` is `True`, the plotted indicator marker will be a
         circle instead of a connected line segment (default).
 
+        `legends` can be list or array of string values to represent
+        legends on your indicator chart. By default it's set to None,
+        and `name` is used as legends.
```

Comment:
Why don't we simply reuse `name=` to accept a list of name values?

Reply:
I didn't use it thinking it may be used in other places. so I wanted to keep it backward compatible. But if it's not being used elsewhere, I can change my code to use name instead of introducing a new attribute. Let me know.

Reply:
@kernc did you get a chance to look at this?

Reply:
I am pretty sure, it should be possible to reuse `name=`, since `color=` is used in the same fashion. For instance here is an [example](https://github.com/kernc/backtesting.py/discussions/419#discussioncomment-1033981) how to set multiple colors.

Having two arguments to set one thing is very confusing from the API perspective.

Reply:
@ivaigult Can you try and let me know. My first attempt was to pass array to name but it didn't work. I already mentioned my reasoning in previous comment. If you can make it work with name that would be great :)

Reply:
@zlpatel I uploaded my attempt in #980 :slightly_smiling_face: